docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -751,3 +751,51 @@ spec:
   tags: compliance, hardening, built-in
   platform: darwin
   contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Operating system up to date (macOS)
+  query: SELECT 1 FROM os_version WHERE version >= '12.5.1';
+  description: "Checks that the operating system is up to date."
+  resolution: "Go to the Software Update section of System Preferences and install the latest updates."
+  platforms: macOS
+  tags: compliance, CIS, template, CIS1.1
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Automatic updates enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticCheckEnabled' AND value=1 LIMIT 1;
+  description: "Checks that the operating system is configured to automatically checks for updates."
```

```suggestion
  description: "Checks that a mobile device management (MDM) solution configures the operating system to automatically checks for updates."
```

I see that for other queries that check `managed_policies`  we specify that the policy checks if MDM is doing the configuring. 

Here's an example policy where we specify this: https://fleetdm.com/queries/screen-lock-enabled-via-mdm-profile-mac-os

I think we do this because it's possible that the operating system is configured differently than what the MDM enforces. Currently, I think we have no way to detect this.

If this is still the case, I suggest applying the same "Checks that a mobile device management (MDM) solution configures..." to this policy's description.